python/cuspatial/CMakeLists.txt

```diff
@@ -40,22 +42,54 @@ else()
 endif()
 
 if(NOT cuspatial_FOUND)
-  # TODO: This will not be necessary once we upgrade to CMake 3.22, which will
-  # pull in the required languages for the C++ project even if this project
-  # does not require those languages.
-  include(rapids-cuda)
-  rapids_cuda_init_architectures(cuspatial)
-  enable_language(CUDA)
-  # Since cuspatial only enables CUDA optionally we need to manually include the file that
-  # rapids_cuda_init_architectures relies on `project` including.
-  include("${CMAKE_PROJECT_cuspatial_INCLUDE}")
-
-  add_subdirectory(../../cpp cuspatial-cpp)
-
-  install(TARGETS cuspatial DESTINATION cuspatial/_lib)
+  set(BUILD_TESTS OFF)
+  set(BUILD_BENCHMARKS OFF)
+  set(_exclude_from_all "")
+  if(CUSPATIAL_BUILD_WHEELS)
+
+    # We don't build C++ tests when building wheels, so we can also omit the test util and shrink
+    # the wheel by avoiding embedding GTest.
+    set(CUDF_BUILD_TESTUTIL OFF)
+    set(CUDF_BUILD_STREAMS_TEST_UTIL OFF)
```

Comment:
These are redundant because you're setting `BUILD_TESTS OFF` above and that is directly forwarded to the cudf build AFAICT.

Reply:
Why is build tests off?

Reply:
We don't want to build test into the wheel, and we certainly don't want to build libcudf tests into a cuspatial python wheel.